datanommer.models/datanommer/models/__init__.py

```diff
@@ -114,6 +120,12 @@ def init(uri=None, alembic_ini=None, engine=None, create=False):
             command.stamp(alembic_cfg, "head")
 
 
+def refresh(mapper, connection, target):
```

Comment:
If it proves too much load to refresh the view on each insert, I would like to have a command that I would be able to run in cron every 10 minutes or every hour to run the `REFRESH` command. Could you pleae make a function that would be more easily callable on it own, only taking the connection as an argument?
The event callback could then be `lambda mapper, connection, target: refresh(connection)`.